<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Metadata rewrite rules:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rewrite_rules_combo"/>
     </item>
     <item>
      <widget class="QPushButton" name="rewrite_rules_edit_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Folder tags from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="root_input"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Under:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="root_tag_input"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Extra tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="extra_tags_input"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Extra metadata:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="metadata_view"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="preview"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
